cypress.json

```diff
@@ -1,4 +1,5 @@
 {
-  "baseUrl": "http://localhost:8080",
-  "video": false
+  "baseUrl": "http://localhost:3000",
```

Comment:
Can we not change this port?

Reply:
In this case we need to replace PORT in .env on 8080

Reply:
https://docs.cypress.io/guides/references/configuration.html#Command-Line we can pass a `baseUrl` from command line and pass `PORT` env from there